app/models/event.rb

```diff
@@ -17,4 +18,22 @@ def before_status_full
   def after_status_full
     STATUSES[after_status]
   end
+
+  def new_for?(user)
+    notifications.where('created_at > ? and user_id = ?',
+                        user.last_check_notifications_at, user.id).present?
+  end
+
+  def unread_by?(user)
+    notifications.unread.find_by(user_id: user.id).present?
+  end
+
+  private
+
+  def create_notifications
+    users = issue.followers(User)
+    users.each do |u|
```

Comment:
this is OK for now .... but on real project it can be a problem ... what if you have 100 followers, then you need to create a lot of notifications.
such tasks need to be done 
1) in background
2) batch insert into DB